mindmeld/components/nlp.py

```diff
@@ -1134,6 +1134,26 @@ def get_entity_processors(self, label_set=None):
 
     def _build(self, incremental=False, label_set=None, load_cached=True):
         """Builds the models for this intent"""
+        entity_types = self.entity_recognizer.get_entity_types(label_set=label_set)
```

Comment:
Isn't this just getting overwritten by line 1170?

Reply:
Yeah, it gets overwritten. Missed to remove this line from previous edition of code created as part of PR 325. Removed this line now. Thanks for pointing it out.